R/ppc-intervals.R

```diff
@@ -173,14 +173,17 @@ ppc_intervals_grouped <- function(y,
 
 #' @rdname PPC-intervals
 #' @export
+#' @param y_draw For ribbon plots only, a string specifying how to draw `y. Can
+#'   be `"line"` (the default), `"points"`, or `"both"`.
```

Comment:
One reason CI was failing was because of a missing doc entry for the y_draw argument. I added that here.